ascent_base/src/lattice.rs

```diff
@@ -19,7 +19,7 @@ pub trait Lattice: PartialOrd + Sized {
    ///
    /// Returns true if `self` was changed.
    fn meet_mut(&mut self, other: Self) -> bool {
-      let res = !(*self <= other);
+      let res = *self > other;

```

Comment:
This looks wrong. With partial orders, `!(a <= b)` and `a > b` are not equivalent.

Reply:
I've replaced it with an `#[allow(…)]`.